docker-compose.dev.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  redis:
+    image: redis:latest
```

Comment:
It's a better practice to not use latest tag, just use the current latest version tag.